tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/AlfrescoStackInitializer.java

```diff
@@ -0,0 +1,139 @@
+package org.alfresco.elasticsearch;
+
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.context.support.TestPropertySourceUtils;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.lifecycle.Startables;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+
+public class AlfrescoStackInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext>
+{
+
+    public static Network network;
+
+    public static GenericContainer alfresco;
+
+    public static ElasticsearchContainer elasticsearch;
+
+    public static GenericContainer liveIndexer;
+
+    private static String ES_CONNECTOR_TAG = "3.0.0-RC1";
```

Comment:
This version doesn't match the one in the pom file.  Is it possible for us to define the ESC version once and use it everywhere?

Reply:
solved: .env is loaded